lib/ts/recipe/emailpassword/emailPasswordAuth.tsx

```diff
@@ -35,24 +35,32 @@ type Props = FeatureBaseProps & {
 
 function EmailPasswordAuth(props: Props) {
     const emailVerification = (
-        <EmailVerificationAuth recipe={props.recipe.emailVerification} history={props.history}>
+        <EmailVerificationAuthWrapper
+            recipe={props.recipe.emailVerification}
+            history={props.history}
+            userContext={undefined}>
             {props.children}
-        </EmailVerificationAuth>
+        </EmailVerificationAuthWrapper>
     );
 
     if (props.requireAuth === false) {
-        return <SessionAuth onSessionExpired={props.onSessionExpired}>{emailVerification}</SessionAuth>;
+        return (
+            <SessionAuthWrapper onSessionExpired={props.onSessionExpired} userContext={undefined}>
```

Comment:
Why do you need to pass `userContext={undefined}` explicitly?

Reply:
Added a comment to explain it (also added a comment in other places to refer to this one)

Reply:
Removed passing userContext as undefined everywhere. We only needed it in EmailVerification originally but that has been refactored to use a Consumer instead so not needed there anymore as well.

Reply:
sounds good.